docs/index.md

```diff
@@ -0,0 +1,339 @@
+---
+hide:
+  - navigation
+---
+
+<style>
+.md-content .md-typeset h1 { 
+  position: absolute;
+  left: -999px;
+}
+</style>
+
+
+[![Build Status](https://github.com/mobiusml/aana_sdk/actions/workflows/python-package.yml/badge.svg)](https://github.com/mobiusml/aana_sdk/actions/workflows/python-package.yml)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](/LICENSE)
```

Comment:
The licence link is not working here.
I think you can't point to the repo files from the gh-page and need to add full addr

Reply:
Fixed all links